### PR TITLE
Update "Build a subset of the code" section

### PR DESCRIPTION
**Pull request by captainsafia**
Now that we are referring to this section more frequently from the docs, I've given it a spruce up.

### CODE REVIEW

File: docs/BuildFromSource.md [hunk 1]
```diff
@@ -278,9 +278,21 @@ On macOS/Linux:
 
 ### Building a subset of the code
 
-This repository is large. Look for `build.cmd`/`.sh` scripts in subfolders. These scripts can be used to invoke build and test on a smaller set of projects.
+When working in the repository, you'll typically be focused on one project area, such as Blazor and SignalR. In that case, it's more epedient to use the `build.cmd` and `build.sh` that are available in each subfolder. When invoked in a subfolder on Windows:
```
Comment (Tratcher) [inline docs/BuildFromSource.md:281]: epedient ?

File: docs/BuildFromSource.md [hunk 2]
```diff
@@ -278,9 +278,21 @@ On macOS/Linux:
...
+.\build.cmd
+```
+
+Or on macOS or Linux:
+
+```bash
+./build.sh
+```
+
+> :bulb: Before using the `build.cmd` or `build.sh` at the top-level or in a subfolder, you will need to make sure that the dependencies documented above have been installed.
```
Comment (Tratcher) [inline docs/BuildFromSource.md:293]: Link to said dependencies?

**Review by Tratcher — APPROVED** [review]

File: docs/BuildFromSource.md (hunk 1 above)
Comment (captainsafia) [reply docs/BuildFromSource.md:281]: Should've been expedient. Autocorrect, why y do dis?!!?

File: docs/BuildFromSource.md (hunk 2 above)
Comment (captainsafia) [reply docs/BuildFromSource.md:293]: Good idea!

**Pull request merged**